tools/tools_xml.c

```diff
@@ -49,6 +51,8 @@ crm_mon_xml_begin_list(pcmk__output_t *out, const char *singular_noun, const cha
         name = "bans";
     } else if (!strcmp(buf, "Node Attributes")) {
         name = "node_attributes";
+    } else if (!strcmp(buf, "Resources")) {
+        name = "resources";
     } else if (!strcmp(buf, "Tickets")) {
         name = "tickets";
     } else {
```

Comment:
I'd like to modify this function to get rid of the single word ones and just always call g_ascii_strdown in the else block, but that returns a newly allocated string and I'm not sure if it'll be okay if I free that after creating an XML node with the string.